src/DIRAC/ConfigurationSystem/private/ServiceInterface.py

```diff
@@ -11,31 +11,31 @@
 
 class ServiceInterface(ServiceInterfaceBase, threading.Thread):
     """
-    Service interface, manage Slave/Master server for CS
+    Service interface, manage Worker/Controller server for CS
     Thread components
     """
 
     def __init__(self, sURL):
         threading.Thread.__init__(self)
         ServiceInterfaceBase.__init__(self, sURL)
 
-    def _launchCheckSlaves(self):
+    def _launchCheckWorkers(self):
         """
-        Start loop which check if slaves are alive
+        Start loop which check if Workers are alive
         """
-        gLogger.info("Starting purge slaves thread")
+        gLogger.info("Starting purge Workers thread")
         self.daemon = True
         self.start()
 
     def run(self):
         while True:
             iWaitTime = gConfigurationData.getSlavesGraceTime()
             time.sleep(iWaitTime)
-            self._checkSlavesStatus()
+            self._checkWorkersStatus()
 
     def _updateServiceConfiguration(self, urlSet, fromMaster=False):
```

Comment:
To check if it's ok to change the signature of the method (I assume it is since it's a protected method)
```suggestion
    def _updateServiceConfiguration(self, urlSet, fromController=False):
```